### PR TITLE
Task15

### PR DESCRIPTION
**Pull request by TatsiaA**
task15 | added solution

### CODE REVIEW

File: src/Task15/task15.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace App\Task15;
+
+/**
+ * @param array $arr
+ * @return array
+ */
+function findLostElements(array $arr): array
+{
+    sort($arr);
```
Comment (Jampire) [inline src/Task15/task15.php:13]: Once again, use second argument for sorting if it's not a default sort.

File: src/Task15/task15.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    $arr1 = [];
+    $wholeArrCount = $arr[count($arr) - 1] - $arr[0] + 1;
```
Comment (Jampire) [inline src/Task15/task15.php:15]: It's good implementation, but there is 3 built-in functions to do the same things.

File: tests/Task15/Task15test.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    /**
+     * @test
+     * @dataProvider dataProvider
+     */
+    public function testFindLostElements(array $arr, array $res): void
+    {
+        $this->assertSame($res, findLostElements($arr));
+    }
+
+    public function dataProvider(): array
```
Comment (Jampire) [inline tests/Task15/Task15test.php:22]: Add test case indicating that there is no missing numbers (returns empty array).

**Review by Jampire — CHANGES_REQUESTED** [review]

File: tests/Task15/Task15test.php (hunk 3 above)
Comment (TatsiaA) [reply tests/Task15/Task15test.php:22]: added

File: src/Task15/task15.php (hunk 2 above)
Comment (TatsiaA) [reply src/Task15/task15.php]: better function found

File: src/Task15/task15.php (hunk 1 above)
Comment (TatsiaA) [reply src/Task15/task15.php]: changed
